src/trapezoidal.jl

```diff
@@ -0,0 +1,55 @@
+function __solvebp_call(prob::SampledIntegralProblem, alg::TrapezoidalRule; kwargs...)
```

Comment:
Is there a reason to not use the QuadratureRule for the implementation?

Reply:
No, I am doing that as we speak (although I'm not sure I'll have time to finish today)